lib/make-error.js

```diff
@@ -5,7 +5,7 @@ var errorMessages = {
   'LIMIT_FIELD_KEY': 'Field name too long',
   'LIMIT_FIELD_VALUE': 'Field value too long',
   'LIMIT_FIELD_COUNT': 'Too many fields',
-  'LIMIT_UNEXPECTED_FILE': 'Unexpected field'
+  'LIMIT_UNEXPECTED_FILE': 'Unexpected file'
```

Comment:
Could we change this to "Unexpected file field" or similar since it's the field of the file that is unexpected, not the file itself... thanks!